src/state/store.ts

```diff
@@ -53,7 +53,7 @@ export const createRechartsStore = (
         externalEventsMiddleware.middleware,
         touchEventMiddleware.middleware,
       ]),
-    devTools: {
+    devTools: !!window.RECHARTS_DEV_TOOLS_ENABLED && {
```

Comment:
Can we enable this by default in storybooks?

Reply:
Yes, I've added the flag to the storybook. I hope `preview.ts` is an appropriate place, it looks like config - I haven't worked with storybooks before.